src/components/Profile/History/ComputeJobs/index.tsx

```diff
@@ -79,7 +113,11 @@ export default function ComputeJobs({
         </Button>
       )}
       <Table
-        columns={minimal ? columnsMinimal : columns}
+        columns={
+          minimal
+            ? [columns[5], actionsColumn, columns[4]]
```

Comment:
suggestion: Adding a comment or constant name would help to follow what 5 and 4 do.